planner/core/find_best_task.go

```diff
@@ -43,6 +44,24 @@ const (
 	distinctFactor  = 0.8
 )
 
+var aggFuncFactor = map[string]float64{
+	ast.AggFuncCount:       1.0,
```

Comment:
How do you dicide their values?
And what is `defalut` for?

Reply:
1. The values are how they keep the result of their arguments. For example:
```
avg: avg.sum += expr, avg.cnt += 1
var_pop: var_pop.sum_sqr += expr*expr, var_pop.cnt += 1, var_pop.sum += expr
```

2. The `default` is just for safety.